inst/qml/common/classical/AdvancedOptions.qml

```diff
@@ -44,10 +45,11 @@ Section
 		{
 			name:			"seed"
 			text:			qsTr("Seed")
-			defaultValue:	1
-			min:			1
-			max:			99999999
+			defaultValue:	Math.floor(Math.random() * 1000000) // Init with random integer in [1,...,999999]
```

Comment:
can't this use the setSeed reuseable element?

Reply:
It can, but that does not allow initialisation with a known but random seed.